src/tracing/trace.ts

```diff
@@ -0,0 +1,59 @@
+import * as Sentry from '@sentry/react'
+import { Span, SpanStatusType } from '@sentry/tracing'
+
+interface TraceMetadata {
+  /** Arbitrary data stored on a trace. */
+  data?: Record<string, unknown>
+  /** Indexed (ie searchable) tags associated with a trace. */
+  tags?: Record<string, string | number | boolean>
```

Comment:
if we're allowing these to be anything, can that cause difficulties on the backend later when we have a lot of unmanaged values? i know it will "work", but we chose to type / constrain enum values in `analytics` for a reason - assuming it makes the data set easier to work with on the backend?

Reply:
I think this is a good point, but can be added as a constraint later.

Reply:
constrained through a local typing. good idea!